pkg/machine/config.go

```diff
@@ -135,6 +136,30 @@ type VM interface {
 	Start(name string, opts StartOptions) error
 	State(bypass bool) (Status, error)
 	Stop(name string, opts StopOptions) error
+	SetLock() error
```

Comment:
Why is this a public part of the API? It seems like it should be part of retrieving the machine from disk, not manually called before locking?

Reply:
That's a good point, it probably fits best within `LoadVMByName`